src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfig.tsx

```diff
@@ -395,7 +398,7 @@ export const LimitOrderConfig = ({
               prefix={isInputtingFiatSellAmount ? localeParts.prefix : ''}
               value={
                 isInputtingFiatSellAmount
-                  ? bnOrZero(fiatValue).isZero()
+                  ? !fiatValue || (bnOrZero(fiatValue).isZero() && !priceAmountRef.current)
```

Comment:
ditto readability

Reply:
Done in [c3f212c](https://github.com/shapeshift/web/pull/9052/commits/c3f212c6ff94d2d0a52075cf4ff77ea9c5efc3b3)